<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search results ...</string>
  </property>
  <widget class="QListWidget" name="listSearchResults">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>162</width>
     <height>447</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>493</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>464</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Next 100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrev">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>463</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Previous 100</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listSearchResults</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrev</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>prev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectionChanged()</slot>
  <slot>next()</slot>
  <slot>prev()</slot>
 </slots>
</ui>
